bin/copy-labels.js

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+
+const { LabelHandler } = require('../lib/labels.js');
+
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const padding = ' '.repeat(toolName.length);
+
+const help = `\
+This tool copies labels from source repository or file to destination
+repository or file.
+
+Usage: ${toolName} [--config=path] [--user=user] [--token=token]
+       ${padding} [--src-repo=repo] [--dst-repo=repo] [--src-file=path]
+       ${padding} [--dst-file=path] [--delete] [--update]
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --config   Path to config file
+  --user     github username
+  --token    github access token
+  --src-repo Source repository. e.g. 'metarhia/tools'
+  --dst-repo Destination repository. e.g. 'metarhia/tools'
+  --src-file File to read labels from
+  --dst-file File to write labels to
+  --delete   Delete all labels in destination repository before
```

Comment:
Should we even have this option? Isn't it kinda dangerous? What will happen to labeled issues and PRs when all of the labels are deleted?

Reply:
For example, if you have a repo with a bunch of default labels which won't be used and will be deleted anyway. As for your question, if the label gets deleted, it will be removed from issues and PRs that have them. By default, if the label exists it won't be deleted or modified, but you can specify `--update` option to update existing labels.

Reply:
I think, this tool will be used significantly less often on the new repositories as opposed to existing repositories with lots of existing labels that were used before; thus I don't think we need to include a way to delete just the default labels on the new repositories because it can be done by hand quite easily, but at the same time accidentally using this option on the existing repositories may lead to unlabeling and deleting an important information.

Reply:
I guess since we won't be deleting labels as often as updating them this sounds reasonable though I'd prefer to have a prompt to ask the user whether they really want to delete all labels.

Reply:
@SemenchenkoVitaliy we can also make a request to check if any of the labels, that are to be deleted, are used, and in that case either prompt the user if they are okay with deleting the label, or just avoid deleting the label and print a warning saying that "the label is used somewhere and probably shouldn't be deleted." I, personally, like the latter more, since it doesn't enable the user to do anything potentially dangerous.
Also, in the future, we may probably need to add a way to replace one label with the other, and that can be used when the user gets a message about the label being in use when deleting it.

Reply:
Used labels won't be deleted. Also, there is the `--dry-run` option, which will print actions that will be executed including labels which will be deleted. In case removal cannot be performed it will print that labels and links to issues that have those labels.